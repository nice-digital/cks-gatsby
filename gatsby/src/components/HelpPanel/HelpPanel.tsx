import React from "react";

import { Panel } from "@nice-digital/nds-panel";

export const HelpPanel: React.FC = () => (
	<Panel>
		<h2 className="h3">Help develop CKS</h2>
		<p>
			CKS topics rely on feedback from healthcare professionals, patients
			groups, patients and carers to ensure they are accurate and relevant.
		</p>
		<ul aria-label="How to help develop CKS">
			<li>
				<a href="https://agiliosoftware.com/primary-care/prodigy/">
					Help develop CKS topics
				</a>
			</li>
			<li>
				<a href="https://www.nice.org.uk/cks-feedback">
					Send feedback on CKS topics
				</a>
			</li>
		</ul>
	</Panel>
);
