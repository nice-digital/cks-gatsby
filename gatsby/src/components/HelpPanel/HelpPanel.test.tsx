import React from "react";
import { render } from "@testing-library/react";

import { HelpPanel } from "./HelpPanel";

describe("HelpPanel", () => {
	it("should match snapshot", () => {
		expect(render(<HelpPanel />).container).toMatchSnapshot();
	});
});
