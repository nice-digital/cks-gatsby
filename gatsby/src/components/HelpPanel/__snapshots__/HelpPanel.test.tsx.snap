// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HelpPanel should match snapshot 1`] = `
<div>
  <div
    class="panel"
    data-component="panel"
  >
    <h2
      class="h3"
    >
      Help develop CKS
    </h2>
    <p>
      CKS topics rely on feedback from healthcare professionals, patients groups, patients and carers to ensure they are accurate and relevant.
    </p>
    <ul
      aria-label="How to help develop CKS"
    >
      <li>
        <a
          href="https://agiliosoftware.com/primary-care/prodigy/"
        >
          Help develop CKS topics
        </a>
      </li>
      <li>
        <a
          href="https://www.nice.org.uk/cks-feedback"
        >
          Send feedback on CKS topics
        </a>
      </li>
    </ul>
  </div>
</div>
`;
